CompilerSource/compiler/compile.cpp

```diff
@@ -660,6 +660,7 @@ int lang_CPP::compile(const GameData &game, const char* exe_filename, int mode)
   }
 
   int makeres = e_execs(compilerInfo.MAKE_location, make, flags);
+  if (build_stopping) { build_stopping = false; return 0; }
```

Comment:
If this is the only place you're resetting `build_stopping`, a user can skip it by pressing cancel right as the build ends (or doing any latent process right after that).

Reply:
Alright, ae221fe6a41592d436ad2fb6eabafb53006b8b75 I added an initial reset.

Reply:
Okay, good enough.